target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -133,6 +133,7 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
 		;;
+	sitecom,wlr-4100v1002|\
```

Comment:
alphabetical order

this means changing engenius,esr600 to have the `|\` and your board have the `)`